docs/resources/network.md

```diff
@@ -16,11 +16,26 @@ Network resource schema. Must have a `region` specified in the provider configur
 resource "stackit_network" "example" {
   project_id         = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
   name               = "example-network"
-  nameservers        = ["1.2.3.4", "5.6.7.8"]
+  ipv4_nameservers   = ["1.2.3.4", "5.6.7.8"]
   ipv4_prefix_length = 24
+  ipv4_gateway       = "10.1.2.1"
+  ipv4_prefix        = "10.1.2.0/24"
   labels = {
     "key" = "value"
   }
+  routed = false
+}
+resource "stackit_network" "example_ipv6" {
+  project_id         = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+  name               = "example-network"
+  ipv6_nameservers   = ["2001:4860:4860::8888", "2001:4860:4860::8844"]
+  ipv6_prefix_length = 56
+  ipv6_gateway       = "10.1.2.1"
+  ipv6_prefix        = "10.1.2.0/24"
```

Comment:
These are IPv4 values, right?

Reply:
yes, I took the values from the api examples but we can change them.

Reply:
we have decided to remove the ipv6 example as it is not yet supported.